src/App.vue

```diff
@@ -666,10 +666,11 @@ export default class App extends Mixins(
     // NB: these were already validated in storeBootstrapItem()
     const header = filing.header
     const data = filing[header.name]
+    const status = filing.header.status as FilingStatus
```

Comment:
Since `header` is already de-referenced, you can save a few nanoseconds by using it here instead of de-referencing it again.

Also on line 695.

Reply:
And line 719.

Reply:
Updated